hr_expense_move_date/models/hr_expense.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, fields
+
+
+class HrExpenseExpense(models.Model):
+    _inherit = 'hr.expense.expense'
+
+    move_date = fields.Date(
+        string='Move Date',
+    )
+
+    @api.model
+    def account_move_get(self, expense_id):
+        res = super(HrExpenseExpense, self).account_move_get(expense_id)
+        expense = self.browse([expense_id])[0]
```

Comment:
if changed `expense = self.browse(expense_id)` gives you a record so no need to index into a recordset